cmd/manager/main.go

```diff
@@ -149,16 +147,19 @@ func main() {
 		WithServiceName(operatorconfig.OperatorName).
 		GetConfig()
 
-	// detect if operator-sdk up local is run
-	detectSDKLocal := os.Getenv(envSDK)
-
-	// Configure metrics. If it errors, log the error and exit.
-	if detectSDKLocal == "local" {
-		log.Info("Skipping metrics configuration; not running in a cluster.")
+	// Get the namespace the operator is currently deployed in.
+	if _, err := k8sutil.GetOperatorNamespace(); err != nil {
```

Comment:
Just out of curiosity, why change this from the current pattern?

Reply:
I was having issues getting the metrics-server running locally and these changes were remnants of that troubleshooting. This is mainly to bring it up to date with operator-sdk (https://github.com/operator-framework/operator-sdk/blob/master/website/content/en/docs/upgrading-sdk-version/version-upgrade-guide.md).

Reply:
:+1: